tests/Person.spec.ts

```diff
@@ -459,6 +459,31 @@ test.describe(`Testfälle für die Administration von Personen": Umgebung: ${pro
     });
   });
 
+
+  test("In der Ergebnisliste die Filterfunktion der Schulen benutzen als Landesadmin", {tag: [LONG, SHORT, STAGE]}, async ({ page }) => {
+    const personManagementView = new PersonManagementViewPage(page);
+    const schulstrukturknoten = "Testschule-PW665";
```

Comment:
In the future we should setup data directly in the tests instead of using seeding data :)